### PR TITLE
Prep for landing rhel7/8 containers on the corresponding rhel nodes in 4.1

### PR DESCRIPTION
**Pull request by jmencak**

### CODE REVIEW

**Comment by jmencak** [comment]
WIP, but tested, @sjug PTAL

**Comment by jmencak** [comment]
@sjug CI finally happy, please take a look.

File: pkg/config/config.go [hunk 1]
```diff
@@ -15,8 +15,8 @@ const (
 )
 
 // NodeTunedImage returns the operator's operand/tuned image path.
-func NodeTunedImage() string {
-	nodeTunedImage := os.Getenv("CLUSTER_NODE_TUNED_IMAGE")
+func NodeTunedImage(version string) string {
+	nodeTunedImage := os.Getenv("CLUSTER_NODE_TUNED_IMAGE_" + version)
```
Comment (sjug) [inline pkg/config/config.go]: We might want to limit `version` to only `7|8`?

File: manifests/04-operator.yaml [hunk 2]
```diff
@@ -44,7 +44,7 @@ spec:
             value: "node-tuning"
           - name: RESYNC_PERIOD
             value: "600"
-          - name: CLUSTER_NODE_TUNED_IMAGE
+          - name: CLUSTER_NODE_TUNED_IMAGE_RHEL7
```
Comment (sjug) [inline manifests/04-operator.yaml]: Will the config have both keys `CLUSTER_NODE_TUNED_IMAGE_RHEL7` and `CLUSTER_NODE_TUNED_IMAGE_RHEL8`

File: pkg/controller/tuned/tuned_controller.go [hunk 3]
```diff
@@ -375,7 +375,7 @@ func (r *ReconcileTuned) Reconcile(request reconcile.Request) (reconcile.Result,
 		return reconcileResult, err
 	}
 
-	err = r.syncDaemonSet(tunedInstance)
+	err = r.syncDaemonSet(r.manifestFactory.TunedDaemonSetRHEL7, tunedInstance)
```
Comment (sjug) [inline pkg/controller/tuned/tuned_controller.go]: This hardcoded version instance is also just for now, like line 32 above in `syncOperatorStatus()`?

File: pkg/config/config.go (hunk 1 above)
Comment (jmencak) [reply pkg/config/config.go]: Why?

File: manifests/04-operator.yaml (hunk 2 above)
Comment (jmencak) [reply manifests/04-operator.yaml]: Eventually yes.

File: pkg/controller/tuned/tuned_controller.go (hunk 3 above)
Comment (jmencak) [reply pkg/controller/tuned/tuned_controller.go]: If the decision is made for several daemonsets to be used, then yes, there will be an additional r.manifestFactory.TunedDaemonSetRHEL8

**Comment by jmencak** [comment]
Until it is clear whether 2 daemonsets should really be used or 2 containers one put to sleep.

/hold

**Pull request closed without merging**